Source/cryptography/implementation/OpenSSL/Cipher.cpp

```diff
@@ -111,7 +112,7 @@ class Cipher : public CipherImplementation {
         } else if (maxOutputLength < inputLength) {
             // Note: Pitfall, AES CBC/ECB will use padding
             TRACE_L1(_T("Too small output buffer, expected: %i bytes"), inputLength);
-            result = (-inputLength);
+            result = (~inputLength);
```

Comment:
a) If we change this here, would be good to update the other implementations for consistency
b) this code is too lazy anyway, it needs to take the block cipher mode into account before suggesting the required size due to possible padding

Reply:
It was a warning in the windows build :-) I just made the warning go away and still achieve the same. Agree it is good to align the other implementation as well than.
I do not get point b :-)

Reply:
Oke, after our discussion I do get point B. Will update the pull request accordingly and indeed I stand corrected. The ~ is differetn than -. Will adapt it accordingly.